--- origsrc/qalculate-gtk-0.9.6/configure.in	2007-05-18 03:36:10.000000000 -0500
+++ src/qalculate-gtk-0.9.6/configure.in	2008-02-04 21:47:32.765625000 -0600
@@ -63,7 +63,7 @@
 
 AC_LANG([C++])
 
-AC_PATH_CLN(1.1.0, [
+PKG_CHECK_MODULES(CLN, cln >= 1.1.0, [
      	LIBS="$LIBS $CLN_LIBS"
      	CPPFLAGS="$CPPFLAGS $CLN_CPPFLAGS"
   	AC_DEFINE(HAVE_LIBCLN,1,[If CLN can be used])
@@ -76,7 +76,7 @@
 AC_SUBST(GLIB_CFLAGS)
 AC_SUBST(GLIB_LIBS)
 
-QALCULATE_GTK="qalculate-gtk"
+QALCULATE_GTK="qalculate-gtk\$(EXEEXT)"
 LN_QALCULATE="qalculate-gtk"
 
 PKG_CHECK_MODULES(GTK, [
--- origsrc/qalculate-gtk-0.9.6/src/Makefile.am	2006-07-16 02:21:54.000000000 -0500
+++ src/qalculate-gtk-0.9.6/src/Makefile.am	2008-02-04 21:47:02.484375000 -0600
@@ -25,6 +25,4 @@
 	@LIBGNOME_LIBS@ \
 	@QALCULATE_LIBS@
 
-install-exec-local:
-	cd $(DESTDIR)$(bindir) && rm -f qalculate; $(LN_S) @LN_QALCULATE@ qalculate
 
